models/tasks.js

```diff
@@ -1,6 +1,7 @@
 const firestore = require('../utils/firestore')
 const tasksModel = firestore.collection('tasks')
 const { fetchUser } = require('./users')
+const userMapping = require('../utils/users')
```

Comment:
We can change the name to `userUtils`, or any other descriptive name, as the file does not only contain the mapping functions.

Reply:
ok